.evergreen/config.yml

```diff
@@ -2236,19 +2226,6 @@ buildvariants:
     # Test standalone and sharded only on 5.0 and later.
     - '.5.0'
 
-- matrix_name: "tests-pyopenssl-pypy"
```

Comment:
Testing pypy3.x is merged into the matrix that tests cpython above. I don't know why they were separate.

Reply:
Ack.